t1rocketemu/src/TestBench.scala

```diff
@@ -127,7 +127,7 @@ class TestBench(val parameter: T1RocketTileParameter)
   }
   instFetchAgent.io.clock := clock
   instFetchAgent.io.reset     := reset
-  instFetchAgent.io.channelId := 0.U
+  instFetchAgent.io.channelId := 2.U
```

Comment:
why this is configure with 2?

Reply:
Because the previous setting was wrong and conflicted with one of the ports (iirc highBandwidth) from vLSU